packages/ui/test/working-groups/modals/AddWorkerStakeModal.test.tsx

```diff
@@ -0,0 +1,194 @@
+import { cryptoWaitReady } from '@polkadot/util-crypto'
+import { configure, fireEvent, render, screen } from '@testing-library/react'
+import React from 'react'
+
+import { Account } from '@/accounts/types'
+import { ApiContext } from '@/common/providers/api/context'
+import { ModalContext } from '@/common/providers/modal/context'
+import { AddWorkerStakeModal } from '@/working-groups/modals/AddStakeModal'
+
+import { getButton } from '../../_helpers/getButton'
+import { alice, bob } from '../../_mocks/keyring'
+import { MockKeyringProvider, MockQueryNodeProviders } from '../../_mocks/providers'
+import { setupMockServer } from '../../_mocks/server'
+import {
+  stubApi,
+  stubDefaultBalances,
+  stubTransaction,
+  stubTransactionFailure,
+  stubTransactionPending,
+  stubTransactionSuccess,
+} from '../../_mocks/transactions'
+
+configure({ testIdAttribute: 'id' })
+
+const useMyAccounts: { hasAccounts: boolean; allAccounts: Account[] } = {
+  hasAccounts: true,
+  allAccounts: [],
+}
+
+jest.mock('@/accounts/hooks/useMyAccounts', () => {
+  return {
+    useMyAccounts: () => useMyAccounts,
+  }
+})
+
+describe('UI: AddStakeModal', () => {
+  setupMockServer()
+  const api = stubApi()
+  let transfer: any
+
+  const mockModalContext = {
+    hideModal: jest.fn(),
+    modalData: {
+      worker: {
+        minStake: 300,
+        stake: 100,
+        roleAccount: alice.address,
+        id: 'workerId',
+        group: {
+          id: 'storageWorkingGroup',
+        },
+        runtimeId: 88,
+      },
+    },
+    modal: null,
+    showModal: jest.fn(),
+  }
+
+  beforeAll(async () => {
+    await cryptoWaitReady()
+    useMyAccounts.allAccounts.push(alice, bob)
+  })
+
+  beforeEach(async () => {
+    stubDefaultBalances(api)
+    transfer = stubTransaction(api, 'api.tx.storageWorkingGroup.increaseStake')
+  })
+
+  describe('Form step', () => {
+    it('Renders modal', () => {
+      const { queryByText } = renderModal()
+
+      expect(queryByText('Add stake')).toBeDefined()
+    })
+
+    it('Setup minimal stake', async () => {
+      renderModal()
+
+      const input = await getStakeInput()
+      const submitButton = await getButton('Add Stake')
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore
+      expect(input?.value).toBe('200')
```

Comment:
```suggestion
      expect(input.value).toBe('200')
```
I would strongly encourage you to not do that. This rule suggest that you might be working with wrong type. The TS ignore is the simplest fix – by not looking that the error exists. That's why we also have the ESLint rule for that ;)

A better solution would be just explicitly tell TS that we're sure that this is a `HTMLInputElement` by type assertion: `const input = (await getStakeInput()) as HTMLInputElement` but fortunately, the testing library got us covered 👇

Reply:
Totally forget about type casting. Thanks for improvement!